build.gradle.kts

```diff
@@ -94,19 +101,38 @@ allprojects {
 
     plugins.withType<JavaPlugin> {
         configure<JavaPluginExtension> {
-            sourceCompatibility = JavaVersion.VERSION_1_8
-            targetCompatibility = JavaVersion.VERSION_1_8
             withSourcesJar()
-            if (!skipJavadoc) {
+            if (!skipJavadoc && false) {
```

Comment:
You can add `skipJavadoc=true` to your `$HOME/.gradle/gradle.properties` by the way.